test/Spaces/ddss1.jl

```diff
@@ -61,41 +71,54 @@ end
     @test Topologies.local_neighboring_elements(space.topology, 3) == [2, 4]
     @test Topologies.local_neighboring_elements(space.topology, 4) == [1, 3]
 
-    init_state(local_geometry, p) = (ρ = 1.0)
-    y0 = init_state.(Fields.local_geometry_field(space), Ref(nothing))
+    y0 = init_state_scalar.(Fields.local_geometry_field(space), Ref(nothing))
     nel = Topologies.nlocalelems(Spaces.topology(space))
     yarr = parent(y0)
     yarr .= reshape(1:(Nq * Nq * nel), (Nq, Nq, 1, nel))
 
-    dss2_buffer = Spaces.create_dss_buffer(y0)
-    Spaces.weighted_dss!(y0, dss2_buffer) # DSS2
+    dss_buffer = Spaces.create_dss_buffer(y0)
+    Spaces.weighted_dss!(y0, dss_buffer) # DSS2
 #! format: off
     @test Array(yarr[:]) == [18.5, 5.0, 9.5, 18.5, 5.0, 9.5, 18.5, 5.0, 9.5, 9.5, 
                              14.0, 18.5, 9.5, 14.0, 18.5, 9.5, 14.0, 18.5, 18.5, 
                              23.0, 27.5, 18.5, 23.0, 27.5, 18.5, 23.0, 27.5, 27.5, 
                              32.0, 18.5, 27.5, 32.0, 18.5, 27.5, 32.0, 18.5]
 #! format: on
 
-    p = @allocated Spaces.weighted_dss!(y0, dss2_buffer)
+    p = @allocated Spaces.weighted_dss!(y0, dss_buffer)
     @show p
     #=
     @test p == 0
     =#
 end
 
-@testset "4x1 element mesh on 2 processes - vector field" begin
+@testset "test if dss is no-op on an empty field" begin
+    Nq = 3
+    space, comms_ctx = distributed_space((4, 1), (true, true), (Nq, 1, 1))
+    y0 = init_state_scalar.(Fields.local_geometry_field(space), Ref(nothing))
+
+    dims = (Nq, Nq, 0, 4)
+    array = similar(parent(y0), dims)
+    data = DataLayouts.rebuild(Fields.field_values(y0), array)
+    space = axes(y0)
+    empty_field = Fields.Field(data, space)
```

Comment:
Should be easier to do
```suggestion
    empty_field = similar(y0, Tuple{})
```